src/css/index.css

```diff
@@ -131,3 +131,57 @@ pre {
         opacity: 0.9;
     }
 }
+
+.sets-manager .current-set-manager {
```

Comment:
We have enough potential nesting here that sass or less might be worthwhile.

Reply:
I have added the dependency `nwb-sass` which was documented by nwb https://github.com/insin/nwb/blob/master/docs/Stylesheets.md